ppl/zqd/apiserver/manager.go

```diff
@@ -39,15 +39,10 @@ type Manager struct {
 	deleted prometheus.Counter
 }
 
-func NewManager(ctx context.Context, logger *zap.Logger, registerer prometheus.Registerer, root iosrc.URI, dbconf db.Config) (*Manager, error) {
-	db, err := db.Open(ctx, logger, dbconf, root)
-	if err != nil {
-		return nil, err
-	}
-
+func NewManager(ctx context.Context, logger *zap.Logger, registerer prometheus.Registerer, root iosrc.URI, db db.DB) (*Manager, error) {
 	factory := promauto.With(registerer)
 	m := &Manager{
-		logger:   logger,
+		logger:   logger.Named("manager").With(zap.String("root", root.String())),
```

Comment:
I don't think the root needs to be included on every manager log, how about adding it to the `Loaded` info line below once?

Reply:
Sounds good.